# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../clanlib06/freetype_fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./Sources/TTF/font_ttf.cpp.orig	2005-02-23 01:05:03.503544152 -0300
+++ ./Sources/TTF/font_ttf.cpp	2005-02-23 01:05:55.267674808 -0300
@@ -21,7 +21,8 @@
 #include <malloc.h>
 #include "Display/Font/font_generic.h"
 
-#include <freetype/freetype.h>
+#include <ft2build.h>"
+#include FT_FREETYPE_H
 
 #include "font_ttf.h"
 
